ui/page/transaction/transactions_page.go

```diff
@@ -90,9 +105,8 @@ func NewTransactionsPage(l *load.Load) *TransactionsPage {
 // Part of the load.Page interface.
 func (pg *TransactionsPage) OnNavigatedTo() {
 	pg.ctx, pg.ctxCancel = context.WithCancel(context.TODO())
-
 	pg.refreshAvailableTxType()
-	if !pg.WL.SelectedWallet.Wallet.IsSynced() {
+	if !pg.selectedWallet.IsSynced() {
```

Comment:
Out of scope for this PR, but what if the wallet becomes synced while this page is open? Can leave a TODO if necessary.